tensorflow/compiler/xla/client/computation_builder.h

```diff
@@ -211,9 +211,11 @@ class ComputationBuilder {
   //
   // Note that "limit" means up-to-but-not-including; i.e. [start, limit) in 1D
   // range notation.
+  // The stride parameter determines the stride over the slice
```

Comment:
Personally I would rather stick to positive only strides (on the grounds of similarity with our backend API).   I suppose that if the 'rev plus slice from end+1 -> begin+1'  code goes into tf2xla then that makes for simpler code in all backends.

I put in negative strides because that's what TF core does.  It does it (probably) because of numpy.